### PR TITLE
Support ElasticSearch in CrossDC integration test

### PR DESCRIPTION
**Pull request by vancexu**
This PR also include XDC integration test of SearchAttributes

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -27,7 +27,7 @@ BUILD := ./build
 TOOLS_CMD_ROOT=./cmd/tools
 INTEG_TEST_ROOT=./host
 INTEG_TEST_DIR=host
-INTEG_TEST_XDC_ROOT=./hostxdc
+INTEG_TEST_XDC_ROOT=./host/hostxdc
```
Comment (yycptt) [inline Makefile]: I think /host/xdc looks better than /host/hostxdc. What do you think?

File: host/util.go [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+import (
+	"github.com/olivere/elastic"
+	"github.com/stretchr/testify/suite"
+	"io/ioutil"
+	"time"
+)
+
+// CreateESClient create ElasticSearch client for test
+func CreateESClient(s suite.Suite, url string) *elastic.Client {
```
Comment (yycptt) [inline host/util.go]: Naming of this file is a little bit confusing to me. How about moving these methods to elasticsearch_test.go? or change the file name to something like es_util.go?

File: Makefile (hunk 1 above)
Comment (vancexu) [reply Makefile]: updated

File: host/util.go (hunk 2 above)
Comment (vancexu) [reply host/util.go]: I would keep it. 
The purpose is to put shared code for host and hostxdc in one place. I expect more shared code (not only related to es) in future, which should live together.

**Pull request merged**